spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java

```diff
@@ -128,6 +124,15 @@ public void setApplicationContext(ApplicationContext context) throws BeansExcept
 		return builder;
 	}
 
+	private Logger getLogger(FeignContext context) {
+		Logger logger = getOptional(context, Logger.class);
```

Comment:
Well, we could do that, but, I'm thinking if that would really be a "default" implementation then. I was thinking of the default implementation to be more of a fallback mechanism than the core logic (of identifying the logger implementation). What do you think?